scanpy/preprocessing/_pca.py

```diff
@@ -22,6 +22,8 @@ def pca(
     random_state: AnyRandom = 0,
     return_info: bool = False,
     use_highly_variable: Optional[bool] = None,
+    use_existing_mask: Optional[str] = None,
+    mask: Optional[np.ndarray] = None,
```

Comment:
As discussed, I think these can be collapsed into a single argument. I've expanded on this in the body of the review.